game/assets/jak1/text/game_text_en.gs

```diff
@@ -437,6 +437,8 @@
 (#x1524 "ALL CUTSCENES"
         "ALL CUTSCENES")
 
+;; Custom Level Text: 1600-1800
+
```

Comment:
this is a fairly fragile thing to depend on. for example this PR adds things in the `1600` range https://github.com/open-goal/jak-project/pull/2397 

If you want to reserve a range, you should actually add the entries to the enum so we don't touch them.  I struggle to see how this will work end-to-end consistency but that's a separate issue.

Reply:
I'm not trying to add things here manually. Blender will be doing it automatically. I can't add them ahead of time unless they had obnoxious placeholders which would be... worse than nothing

Reply:
You can add an entry to the actual enum at 1600 and 1800 so people know not to add something in that range.  I think you still have a ton of issues to solve related to that but atleast then when we go to add something in `text-h.gc` it's very obvious we should avoid that range.

I'm just saying a comment in the english text file isn't going to guarantee anything for you.